### DISCUSSION TITLE
How to use "trim version string" option?

### DISCUSSION THREAD

**Discussion by BBaoVanC** [ImranR98/Obtainium#1781] [Q&A]
I couldn't find any documentation on how to use this option. I'm installing Home Assistant which adds a `-full` suffix to the version (`2024.7.3-full`) because I have the version that requires Play Services. The GitHub tag just has the version (`2024.7.3`). Is this what the trim version string feature is intended to do? I tried setting the regex to `(.*)-full` and setting the match group to `$1`, but it just says "Could not determine release version".

I know it works by just leaving it with "a pseudo-version is in use", but it annoys me that it's not truly parsing it right.

**Comment by ImranR98** [comment]
This feature is meant for the opposite scenario - when the GitHub tag needs to be trimmed. We never mess with the OS-provided version because it can never be wrong.

**Reply by BBaoVanC** [reply]
Oh, is there a way to add to the GitHub version string (add the `-full` to the end) to match the OS version?

**Reply by callmeTayyy** [reply]
You can! I got it working by setting:
* Trim version as `([0-9.]+)`
* And Match group as `$1-full`
